src/Microsoft.TemplateEngine.Cli/TemplateInvoker.cs

```diff
@@ -106,6 +107,39 @@ private static string GetChangeString(ChangeKind kind)
             };
         }
 
+        private static string AdjustReportedPath(string targetPath, string? requestedOutputPath)
+        {
+            if (string.IsNullOrEmpty(requestedOutputPath))
+            {
+                return targetPath;
+            }
+
+            return TryGetRelativePath(Path.Combine(requestedOutputPath, targetPath), Directory.GetCurrentDirectory());
```

Comment:
do we need this handling?
Will just `Path.Combine(requestedOutputPath, targetPath)`  work?

Reply:
`Path.Combine` might create ugly paths similar to `foo/bar/.\target\file.cs` - so normalization is beneficial.

Test cases are on the way